app/assets/stylesheets/widgets/home_page/blog_stories.scss

```diff
@@ -61,7 +61,7 @@
       }
       .source a:hover, a.tag:hover {
         opacity: 0.8;
-        color: #98002e;
+        color: #a50034;
```

Comment:
Still seeing the hardcoded hex code in this file instead of `$cu-red`. All instances of `#a50034` in SCSS files should be able to use the SCSS variable, unless you test it and find otherwise for whatever reason.